pkg/v2/delegate.go

```diff
@@ -36,6 +37,7 @@ type DelegateConfig struct {
 	OffchainKeyring              types.OffchainKeyring
 	OnchainKeyring               types.OnchainKeyring
 	LocalConfig                  types.LocalConfig
+	MetricsRegisterer            prometheus.Registerer
```

Comment:
Could you create a corresponding PR in core which populate this?

Reply:
@infiloop2 Yeah, I'll create the corresponding PR